lib/core/versions/latest/TransactionSelector.ts

```diff
@@ -66,8 +66,12 @@ export default class TransactionSelector implements ITransactionSelector {
     let numberOfOperations = 0;
     if (transactions) {
       for (const transaction of transactions) {
-        const numOfOperationsInCurrentTransaction = AnchoredDataSerializer.deserialize(transaction.anchorString).numberOfOperations;
-        numberOfOperations += numOfOperationsInCurrentTransaction;
+        try {
+          const numOfOperationsInCurrentTransaction = AnchoredDataSerializer.deserialize(transaction.anchorString).numberOfOperations;
+          numberOfOperations += numOfOperationsInCurrentTransaction;
+        } catch {
+          console.debug(`Unable to deserialize ${transaction.anchorString}, transaction not considered as selected.`);
```

Comment:
Please include the exception details as it might be good to know what happened. You can capture the relevant details of the exception ````e```` with: ````JSON.stringify(e, Object.getOwnPropertyNames(e))````